shell/platform/android/platform_view_android.cc

```diff
@@ -301,10 +301,20 @@ void PlatformViewAndroid::RegisterExternalTexture(
     int64_t texture_id,
     const fml::jni::ScopedJavaGlobalRef<jobject>& surface_texture) {
   if (android_context_->RenderingApi() == AndroidRenderingAPI::kOpenGLES) {
-    RegisterTexture(std::make_shared<AndroidExternalTextureGL>(
-        texture_id, surface_texture, jni_facade_));
+    if (android_context_->GetImpellerContext() != nullptr) {
```

Comment:
I think we'd probably just write it like
```suggestion
    if (android_context_->GetImpellerContext()) {
```

Reply:
Done.